pulumi/aws/config/Pulumi.stackname.yaml.example

```diff
@@ -98,7 +98,7 @@ config:
   # for details.
 
   # Prometheus Configuration
-  prometheus:chart_version: 14.13.1
+  prometheus:chart_version: 14.6.0
```

Comment:
Why did we downgrade Prometheus?

Reply:
We didn't - I had a typo in the initial configuration and that version wasn't valid:

```
prometheus-community/prometheus                         14.6.0          2.26.0          Prometheus is a monitoring system and time seri...
```

Is the current newest.